libsrc/telestrat/ch376.s

```diff
@@ -0,0 +1,344 @@
+
+; jede jede@oric.org 2017-01-22
+
+	; For XA65 compatibily in the future
+    .FEATURE 	c_comments,labels_without_colons,pc_assignment, loose_char_term
```

Comment:
On the Oric, we are using xa65. I did this to share code between cc65 and C compiler in OSDK. Removed here.